drivers/storage/portworx/deployment.go

```diff
@@ -1295,9 +1295,39 @@ func (t *template) getVolumeMounts() []v1.VolumeMount {
 	for _, fn := range extensions {
 		volumeInfoList = append(volumeInfoList, fn()...)
 	}
+	volumeInfoList = t.overrideVolumeMount(volumeInfoList)
 	return t.mountsFromVolInfo(volumeInfoList)
 }
 
+func (t *template) overrideVolumeMount(volInfos []volumeInfo) []volumeInfo {
+	volPaths := map[string]int{}
+
+	for i := range volInfos {
+		volPaths[volInfos[i].mountPath] = i
+	}
+
+	for i := range t.cluster.Spec.Volumes {
```

Comment:
Looks like we'll take the `volInfos` first, then we'll _append_ all the previous `cluster.Spec.Volumes`

This seems to be clashing with how the px-runc is working on the "lower level":
* "default" mounts "go in" first
* "custom mounts" will go after the "default mounts" -- and have the opportunity to override the defaults

Is this code behaving the same?

Reply:
the ticket requires allowing to existing mount paths with a custom hostpath
this function is triggered at last as this will allow us to add more mount paths as well as modify any existing mount paths that is utilized by our application